xla/service/while_loop_unroller.cc

```diff
@@ -65,9 +65,9 @@ namespace {
 using hlo_query::ContainsInstrWithOpcode;
 
 // Parameters for the unroller that can be adjusted.
-const int kUnrollTripCountThreshold = 64;
```

Comment:
I don't think you need this const values anymore.
Please use the properties of the UnrollConfig instead.

Reply:
I did not do this as it requires routing the config through the pass which is significant changes to the pass.

Reply:
I don't think we want to have the global variables.

Reply:
This doesn't seem maintainable and creates hidden state for the `WhileLoopUnroller`. I don't think we should accept the PR in this state.

Can you explain what is the significant difficulty with passing it through the pass? Presumably, you already have access to this config through the `WhileLoopUnroller` itself, so it'd be a matter of passing it around only to helper functions---of which there seems to be `7` (assuming you need it everywhere). Or did I miss something?